vnext/local-cli/generator-windows/templates/metro.config.js

```diff
@@ -37,7 +38,7 @@ module.exports = {
       ),
```

Comment:
We can remove the above blacklisting logic now.

Reply:
No we can't remove the blacklist, because we'll have two copies of react-native (one in node_modules/react-native, and the other in react-native-installation).

---
In reply to: [405209966](https://github.com/microsoft/react-native-windows/pull/4522#discussion_r405209966) [](ancestors = 405209966)